finality-aleph/src/sync/service.rs

```diff
@@ -706,6 +708,17 @@ where
         };
     }
 
+    fn report_sync_status(&mut self) {
+        let prev_status = self.major_sync_last_status;
+        let new_status = self.handler.major_sync();
+        match (prev_status, new_status) {
+            (false, true) => info!(target: LOG_TARGET, "Switched to major-sync mode."),
+            (true, false) => info!(target: LOG_TARGET, "Disabled major-sync mode."),
```

Comment:
It's not really a "mode", since the behaviour of the protocol is identical, perhaps "Switched to major sync state." and "No longer in major sync state." would be better?

Reply:
I'd also suggest renaming the method to `report_sync_status_change`, since it does nothing most of the time.